test/content-proxy.test.js

```diff
@@ -30,15 +30,22 @@ describe('Content Proxy Tests', () => {
   before(async () => {
     nock('https://adobeioruntime.net')
       .get((uri) => uri.startsWith('/api/v1/web/helix'))
-      .reply((uri) => {
+      .reply(async function cb(uri) {
```

Comment:
hm, isn't that the same as:
```
.reply(async (uri) => {
  ...
}
```
don't we usually prefer the arrow notation?

Reply:
yes. but the request headers are only available via `this.req.headers`

Reply:
haha, right, I forgot about the `this` in an arrow function 😉